File: src/mobile/ios/BundlePath.m
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(BundlePath, NSObject)

RCT_EXTERN_METHOD(getBundlePath:(RCTPromiseResolveBlock)resolve
                  rejecter:(RCTPromiseRejectBlock)reject)

@end

File: src/mobile/ios/BundlePath.swift
import Foundation
import React

@objc(BundlePath)
class BundlePath: NSObject {
  @objc func getBundlePath(_ resolve: RCTPromiseResolveBlock,
                           rejecter reject: RCTPromiseRejectBlock) {
    resolve(Bundle.main.bundlePath)
  }

  @objc static func requiresMainQueueSetup() -> Bool { false }
}
